tsconfig.base.json

```diff
@@ -0,0 +1,41 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+    "target": "ES2015",
+    "lib": ["es2017", "dom"],
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "skipDefaultLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "strictFunctionTypes": false,
+    "moduleResolution": "node",
+    "jsx": "preserve",
+    "forceConsistentCasingInFileNames": true,
+    "typeRoots": ["node_modules/@types","./typings"],
+    "baseUrl": ".",
+    "rootDir": ".",
+    "paths": {
+      "@fluentui/react-button": ["packages/react-button/src/index.ts"],
```

Comment:
Can this be shorted to `"@fluentui/*": ["packages/*/src/index.ts"]`, with overrides for cases where that path is incorrect? `tsconfig.fluentui.json` does something similar today.

Reply:
yeah it can but it doesn't conform requirements of  my proposal:
- will we able to use this for convergence only? - no

besides that
- is it explicit ? - not really